app/uk/gov/hmrc/agentclientrelationshipsfrontend/views/client/DeclineRequestPage.scala.html

```diff
@@ -69,9 +69,9 @@ <h1 class="govuk-heading-xl">@pageTitle</h1>
           classes = "govuk-fieldset__legend--m"
         ))
       )),
-      items = Seq("true", "false").map(trueOrFalse =>
+      items = Seq("true", "false").zipWithIndex.map((trueOrFalse,index) =>
         RadioItem(
-          id = Some(DeclineRequestFieldName),
+          id = Some(s"$DeclineRequestFieldName-$index"),
```

Comment:
instead of setting this to a custom id I would recommend removing the setting of the id altogether as using `.withFormField` automates this - because the id was previously being set to `id = Some(DeclineRequestFieldName)` this was erroneously overriding the automatic ids and hardcoding the same fixed id into each radio making the bug